features/docs/stap-definities.feature

```diff
@@ -0,0 +1,326 @@
+      | persoon      | INSERT INTO public.lo3_pl_persoon(stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5)                | 0,0,P,000000012,Jansen |
+
+  Rule: En de persoon is gewijzigd naar de volgende gegevens
+
+    Scenario: Gewijzigd persoon
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam                  | waarde |
+      | geslachtsnaam (02.40) | Jansen |
+      En de persoon is gewijzigd naar de volgende gegevens
+      | naam                  | waarde    |
+      | geslachtsnaam (02.40) | Pietersen |
+      Dan zijn de gegenereerde SQL statements
+      | key          | text                                                                                                                                     | values                 |
+      | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT MAX(pl_id)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING * | 0                      |
+      | persoon      | INSERT INTO public.lo3_pl_persoon(stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5)                | 0,1,P,000000012,Jansen |
+      |              | INSERT INTO public.lo3_pl_persoon(stapel_nr,volg_nr,persoon_type,geslachts_naam) VALUES($1,$2,$3,$4)                                     | 0,0,P,Pietersen        |
+
+  Rule: En de persoon is gecorrigeerd naar de volgende gegevens
+
+    Scenario: Gecorrigeerd persoon
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende gegevens
+      | naam                  | waarde |
+      | geslachtsnaam (02.40) | Jansen |
+      En de persoon is gecorrigeerd naar de volgende gegevens
+      | naam                  | waarde  |
+      | geslachtsnaam (02.40) | Janssen |
+      Dan zijn de gegenereerde SQL statements
+      | key          | text                                                                                                                                     | values                   |
+      | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT MAX(pl_id)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING * | 0                        |
+      | persoon      | INSERT INTO public.lo3_pl_persoon(stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam,onjuist_ind) VALUES($1,$2,$3,$4,$5,$6) | 0,1,P,000000012,Jansen,O |
+      |              | INSERT INTO public.lo3_pl_persoon(stapel_nr,volg_nr,persoon_type,geslachts_naam) VALUES($1,$2,$3,$4)                                     | 0,0,P,Janssen            |
+
+  Rule: Gegeven de persoon met burgerservicenummer '<bsn>' heeft de volgende '<naam-pl-tabel>' gegevens
+
+    Scenario: Persoon heeft 'inschrijving' gegevens
+      Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende 'inschrijving' gegevens
```

Comment:
dit is een heel vreemde beschrijving, want dit is geen inschrijvingsgegeven. Dit is een kiesrecht gegeven:

Gegeven de persoon met burgerservicenummer '000000012' heeft de volgende 'kiesrecht' gegevens

Reply:
Is dit dan consistent? Bij gegevens die naar de lo3_pl_persoon tabel wordt geschreven zoals naam, geboorte hoeft de categorie te worden genoemd.

Reply:
> Is dit dan consistent? Bij gegevens die naar de lo3_pl_persoon tabel wordt geschreven zoals naam, geboorte hoeft de categorie niet te worden genoemd.

juist wel. je kan zowel persoon, partner ouder1/2 en kind toevoegen in lo3_pl_persoon tabel. Dan noem je dat naar de categorie die je toevoegt, niet naar de lo3_pl_persoon tabel. Je voegt ook niet een kind toe met "de persoon heeft de volgende 'persoon' gegevens", omdat het in de lo3_pl_persoon tabel moet komen.
Naam en geboorte zijn geen categorie maar groep en vallen binnen de categorie (persoon, ouder1/2, partner of kind).

In het BRP domein zijn voornaam en geboortedatum persoonsgegevens (of partnergegevens, kindgegevens, enz.), is geheimhouding een inschrijvingsgegeven en is aanduiding Europees kiesrecht een kiesrechtgegeven.

Reply:
Waarom zie ik in aanschrijfwijze dan de volgende gegeven stappen?

```
Gegeven de persoon met burgerservicenummer '000000401' heeft de volgende 'naam' gegevens
```

Reply:
> Waarom zie ik in aanschrijfwijze dan de volgende gegeven stappen?

Omdat die nog gemaakt is met de oude stappen en nog niet goed gemaakt voor de nieuwe Gegeven stappen.